### ISSUE TITLE
Autogen fails: SHA mismatch and "libxml/parser.h: No such file or directory"

### ISSUE THREAD

**Issue by slhck** [Homebrew/linuxbrew-core#11787]
- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Linuxbrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

Gist logs: https://gist.github.com/slhck/755730cb9687f51b97d164efb9bed315

(I fixed the brew remote URL in the meantime; same issue.)

### What you were trying to do (and why)

Install / upgrade `autogen`

### What happened (include command output)

```
➜ brew upgrade autogen                                                                                                                                  
==> Upgrading 1 outdated package:
autogen 5.18.12_1 -> 5.18.16
==> Upgrading autogen 
==> Downloading https://linuxbrew.bintray.com/bottles/autogen-5.18.16.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 8703dbf7fd096c01fb4d97d8775efd2b1f11a849cb5322dd5dab88e21587adad
Actual: fe6abc47017f5a5b94a43379c2664b4a15fedc52a1d29eced30018916f101aa0
Archive: /home/werner/.cache/Homebrew/downloads/0c389a3c7d6a591083617c85ccffc2a88ed9ab74218ace7fdef218268d735f95--autogen-5.18.16.x86_64_linux.bottle.tar.gz
To retry an incomplete download, remove the file above.
Warning: Bottle installation failed: building from source.
==> Downloading https://ftp.gnu.org/gnu/autogen/rel5.18.16/autogen-5.18.16.tar.xz
Already downloaded: /home/werner/.cache/Homebrew/downloads/0c637bd42f4e4f305237e6c652c215ebb1ddcdddc34acf4ab2cdc43f133205ec--autogen-5.18.16.tar.xz
==> ./configure ac_cv_func_utimensat=no --disable-silent-rules --prefix=/home/linuxbrew/.linuxbrew/Cellar/autogen/5.18.16
==> make
Last 15 lines from /home/werner/.cache/Homebrew/Logs/autogen/02.make:
echo '#include "autoopts/project.h"' ; \
for f in xmlopts.c fork.c xml2ag.c ; \
do echo "#include \"$f\"" ; done
gcc-8 -DHAVE_CONFIG_H -I. -I..  -I.. -I../autoopts -I/usr/include/libxml2  -g -O2 -Wno-format-contains-nul -fno-strict-aliasing -Wall -Werror -Wcast-align -Wmissing-prototypes -Wpointer-arith -Wshadow -Wstrict-prototypes -Wwrite-strings -Wstrict-aliasing=3 -Wextra -Wno-cast-qual -g -O2 -Wno-format-contains-nul -fno-strict-aliasing -c -o x.o x.c
In file included from xmlopts.c:47,
                 from x.c:6:
xmlopts.h:244:10: fatal error: libxml/parser.h: No such file or directory
 #include <libxml/parser.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:572: x.o] Error 1
make[2]: Leaving directory '/tmp/autogen-20190302-5816-1cb75yl/autogen-5.18.16/xml2ag'
make[1]: *** [Makefile:547: all-recursive] Error 1
make[1]: Leaving directory '/tmp/autogen-20190302-5816-1cb75yl/autogen-5.18.16'
make: *** [Makefile:454: all] Error 2

READ THIS: https://github.com/Linuxbrew/brew/wiki/troubleshooting
```

### What you expected to happen

I expected `autogen` to either pour the bottle correctly, or build. I do have the correct libraries (`libxml2-dev`) installed:

```
-rw-r--r-- 1 root root 39717 Aug 16  2018 /usr/include/libxml2/libxml/parser.h
```

I have also tried removing the file mentioned above in "To retry an incomplete download, remove the file above.". but it still gives the same SHA error.

**Comment by iMichka** [comment]
Fixed by 5f29980fa6e0c79f2fb9d65e83c5be76d85d0287. I also re-uploaded the bottle.

Please delete `/home/werner/.cache/Homebrew/downloads/0c637bd42f4e4f305237e6c652c215ebb1ddcdddc34acf4ab2cdc43f133205ec--autogen-5.18.16.tar.xz` and try again.

**Comment by slhck** [comment]
Pouring the bottle worked now. Thanks!
